src/isolate.rs

```diff
@@ -347,6 +349,17 @@ impl Isolate {
     unsafe { v8__Isolate__RunMicrotasks(self) }
   }
 
+  /// Optional notification that the system is running low on memory. 
+  /// V8 uses these notifications to attempt to free memory.
+  pub fn low_memory_notification(&mut self) {
+    unsafe { v8__Isolate__LowMemoryNotification(self) }
+  }
+
+  /// Get statistics about the heap memory usage. 
+  pub fn get_heap_statistics(&mut self,  stats:&mut HeapStatistics)  {
```

Comment:
1. I would suggest to change the function signature to `pub fn get_heap_statistics(&self) -> HeapStatistics`. 
(Note that the `self` type also changed from mutable to immutable.)

2. Since this struct is likely to change over time, can you add an assertion that `HeapStatistics` has the same size in Rust and C++. You can find an example how this is done elsewhere [here](https://github.com/denoland/rusty_v8/blob/9ad0d35718673e7afd1ec8d4d6b71ae6c50da60c/src/isolate_create_params.rs#L171-L172).